pkgs/development/python-modules/requests-gssapi/default.nix

```
@@ -0,0 +1,27 @@
+{ fetchPypi
+, lib
+, python3Packages
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "requests-gssapi";
+  version = "1.2.3";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-IHhFCJgUAfcVPJM+7QlTOJM6QIGNplolnb8tgNzLFQ4=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    gssapi
+    requests
+  ];
+  pythonImportCheck = [ "requests_gssapi" ];
```

Upstream uses `pytest` for the unit tests.

```suggestion
  ];

  nativeCheckInputs = [
    pytestCheckHook
  ];

  pythonImportCheck = [ "requests_gssapi" ];
```